web/src/components/studies/StudyDetails.tsx

```diff
@@ -0,0 +1,215 @@
+import styles from "./StudyDetails.module.css";
+import Button from "../ui/Button";
+
+type StudyDetailsProps = {
+  study: Study;
+};
+export default function StudyDetails({ study }: StudyDetailsProps) {
+  const [riskScore, setRiskScore] = useState(0);
+  //   const [feedback, setFeedback] = useState("");
+  const [approvalStatus, setApprovalStatus] = useState("");
+  const [showFeedbackForm, setShowFeedbackForm] = useState(false);
+  const [isCollapsing, setIsCollapsing] = useState(false);
+
+  const handleUpdateStudyStatus = async (status: string) => {
+    const studyId = study.id;
+
+    if (status === "Approved") {
+      // set as approved
+      const response = await postStudiesByStudyIdStatus({ path: { studyId }, body: "Approved" });
+      if (response.response.ok) {
+        setApprovalStatus("Approved");
+      }
+      console.log(response);
+    } else if (status === "Rejected") {
+      // todo: add feedback bits
+    }
+  };
+
+  const toggleShowFeedbackForm = () => {
+    // for smooth collapse
+    if (showFeedbackForm) {
+      setIsCollapsing(true);
+      setTimeout(() => {
+        setShowFeedbackForm(false);
+        setIsCollapsing(false);
+      }, 1900);
+    } else {
+      setShowFeedbackForm(true);
+    }
+  };
+
+  useEffect(() => {
+    const calculateRiskScore = () => {
+      // todo: how to determine if they have data?
+      let score = 0;
+
+      if (study.involves_data_processing_outside_eea) score += 10;
+      if (study.requires_dbs) score += 5;
+      if (study.requires_dspt) score += 5;
+      if (study.involves_third_party && !study.involves_mnca) score += 5;
+      if (study.involves_nhs_england || study.involves_cag) score += 5;
+
+      setRiskScore(score);
+    };
+
+    calculateRiskScore();
+    setApprovalStatus(study.approval_status);
+  }, [study]);
+  return (
+    <>
+      <Box>
+        <div className={styles["pre-description"]}>
+          <span>
+            Last updated: <span className={styles["grey-value"]}>{formatDate(study.updated_at)}</span>
+          </span>
+          <span>
+            Risk Score: <span className={styles["risk-score"]}>{riskScore}</span>
+          </span>
+          <StudyStatusBadge status={approvalStatus} isAdmin={true} />
+        </div>
+        <h2>{study.description}</h2>
+        <div>
+          <dl className={styles.ownership}>
+            <dd>
+              Owner: <span className={styles["grey-value"]}>{study.owner_username}</span>
+            </dd>
+            <dd>
+              Admins:
+              {study.additional_study_admin_usernames.length > 0 && (
+                <span className={styles["grey-value"]}>{study.additional_study_admin_usernames}</span>
+              )}
+            </dd>
+            <dd>
+              Data Controller:{" "}
+              <span className={styles["grey-value"]}>{study.data_controller_organisation.toUpperCase()}</span>
+            </dd>
+          </dl>
+          <h3>Additional Information</h3>
```

Comment:
I vote this is removed in favour of a tooltips showing how much each thing adds to the risk score